regulations/static/package.json

```diff
@@ -30,6 +30,7 @@
   },
   "devDependencies": {
     "@babel/core": "^7.13.16",
+    "@rollup/plugin-alias": "^3.1.8",
```

Comment:
rollup alias plugin added when fixing main.js lint/formatting errors - now we don't have to import from `../../../../../plugin` locations